--- SConstruct	2017-12-12 08:41:10.469790395 -0600
+++ SConstruct	2017-12-12 14:18:59.199747714 -0600
@@ -55,7 +55,7 @@
 # To be used to ensure a PREFIX directory is only created when installing.
 def createPathIsDirCreateWithTarget(target):
   def my_validator(key, val, env):
-    build_targets = (map(str, BUILD_TARGETS))
+    build_targets = (list(map(str, BUILD_TARGETS)))
     if target in build_targets:
       return PathVariable.PathIsDirCreate(key, val, env)
     else:
@@ -166,6 +166,8 @@
                   CPPPATH=['.', ],
                   )

+env.Append( ENV = {'PATH': os.environ['PATH'] } )
+
 env.Append(BUILDERS = {
     'GenDef' :
       Builder(action = sys.executable + ' build/gen_def.py $SOURCES > $TARGET',
